plugins/by-name/grug-far/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, ... }:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "grug-far";
+  originalName = "grug-far.nvim";
+  package = "grug-far-nvim";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  settingsExample = {
+    debounceMs = 1000;
+    minSearchChars = 1;
+    maxSearchMatches = 2000;
+    normalModeSearch = false;
+    searchOnInsertLeave = true;
+    maxWorkers = 8;
+    engine = "ripgrep";
+    engines = {
+      ripgrep = {
+        path = "rg";
+        showReplaceDiff = true;
+      };
+    };
+  };
```

Comment:
I'm assuming it's intentional to have this (excellent) settings example and the defaults test case, but not actually declare any settings sub-options?

Not complaining! Just double checking. :heart_eyes:

Reply:
I just saw the wealth of options they had and don't have the energy to document any atm until I spend any time with the plugin and see what's really useful.

Reply:
I actually prefer not having them as nix options, especially if it seems they may be difficult to understand/maintain.

Reply:
My heuristic is to look at the readme, if they explicitly mention ~5 options I'd add them, else I would not add anything

Reply:
> I'm assuming it's intentional to have this (excellent) settings example and the defaults test case, but not actually declare any settings sub-options?

I see myself in this picture and I don't like it.